src/convertToObject.js

```diff
@@ -1,12 +1,22 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredArr = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item);
+
+  const newArr = filteredArr
+    .map((item) => item.split(':'))
+    .map((item) => item.map((subItem) => subItem.trim()));
```

Comment:
The chained methods should have the same indentation to improve readability. Align the method calls under each other.